test/gtest/ucp/ucp_test.h

```diff
@@ -333,6 +333,9 @@ class ucp_test : public ucp_test_base,
     // Return context parameters of the current test variant
     const ucp_params_t& get_variant_ctx_params() const;
 
+    // Check whether sender and receiver supports certain caps
+    bool supports_caps(uint64_t rndv_modes) const; 
```

Comment:
minor: maybe name it `support_caps` or `is_support_caps`?
It's unusual to have `s` suffix for verbs in function names. E.g. `free_memory`, not `frees_memory`

Reply:
Done.